libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationViewContext.kt

```diff
@@ -25,4 +42,60 @@ internal class DropInNavigationViewContext(
     var routeArrowOptions: RouteArrowOptions = RouteArrowOptions.Builder(context)
         .withAboveLayerId(RouteLayerConstants.TOP_LEVEL_ROUTE_LINE_LAYER_ID)
         .build()
+
+    val onBackPressedCallback = object : OnBackPressedCallback(true) {
+        override fun handleOnBackPressed() = viewModel.onBackPressed()
+    }
+
+    val navigation: MapboxNavigation get() = checkNotNull(MapboxNavigationApp.current()) {
+        "MapboxNavigationApp not initialized. You must call MapboxNavigationApp.setup() first."
+    }
```

Comment:
I think we should delete this

Reply:
Why? This is just an alias with a precondition to ensure that `MapboxNavigationApp.setup()` has been called.